web_listview_range_select/static/src/js/web_listview_range_select.js

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2017 Onestein
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define("web_listview_range_select", function (require) {
+    "use strict";
+
+    var ListRenderer = require("web.ListRenderer");
+
+    ListRenderer.include({
+        /*
+        I extend 'events' because in v13 Odoo catches 'change' event instead
+        of the 'click' event for the selector .o_list_record_selector ",
+        so shift + [click] is not caught.
+        https://github.com/OCA/OCB/blob/13.0/addons/web/static/src/js/views/list/list_renderer.js#L42:L42
```

Comment:
This comment is deprecated, and should change its reference to v14

Reply:
Can I fully remove this comment?

Reply:
In my opinion, that would be the best approach :)

Reply:
done !

Reply:
Thanks, please update the links of the readme file :)

Reply:
oops, my bad, also done